src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -197,6 +197,7 @@ public bool CommitSolutionBinding()
                 {
                     Debug.Assert(this.sourceControlledFileSystem.FileExist(info.NewRuleSetFilePath), "File not written " + info.NewRuleSetFilePath);
                     this.AddFileToSolutionItems(info.NewRuleSetFilePath);
+                    this.RemoveFileFromSolutionItems(info.NewRuleSetFilePath);
```

Comment:
This method name is strange after reading the preceding line. Let's change it to something like `RemoveFileFromOldSolutionItems`